### PR TITLE
Replace mapboxMap with maplibreMap

### PR DESCRIPTION
**Pull request by louwers**
- Remove "mapboxMap" and variations in a bunch of variable names and comments.
- Remove dead code in `CameraUpdateFactory.kt`
- The only potentially API breaking change is the changed FRAG_ARG_MAPLIBREMAPOPTIONS. But I don't the value of this value is persisted between app runs.

### CODE REVIEW

File: platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/annotations/Annotation.java [hunk 1]
```diff
@@ -70,18 +70,18 @@ public void setId(long id) {
   /**
    * Do not use this method, used internally by the SDK.
    *
-   * @param maplibreMap the hosting mapbox map
+   * @param maplibreMap the hosting MapLibreMap
    */
-  public void setMapboxMap(MapLibreMap maplibreMap) {
```
Comment (louwers) [inline platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/annotations/Annotation.java:75]: I mean... The comment is pretty clear right.

Maybe we want to deprecate this first though.

Comment (fynngodau) [reply platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/annotations/Annotation.java:75]: The whole class is deprecated so I'm not sure what use deprecating its individual methods is :)

**Review by cgalvan — APPROVED** [review]

File: platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt [hunk 2]
```diff
@@ -139,7 +139,7 @@ class SymbolGeneratorActivity : AppCompatActivity(), OnMapReadyCallback {
      * Utility class to generate Bitmaps for Symbol.
      *
      *
-     * Bitmaps can be added to the map with [com.mapbox.mapboxsdk.maps.MapboxMap.addImage]
+     * Bitmaps can be added to the map with [MapLibreMap]
```
Comment (mwilsnd) [inline platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt:142]: Was this change fully intentional?

Comment (louwers) [reply platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt:142]: Yes. It works this way.

Comment (louwers) [reply platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt:142]: Oh I see. This is not the same.

Comment (louwers) [reply platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/style/SymbolGeneratorActivity.kt:142]: I don't think this method ever existed. 😛

**Pull request merged**